### DISCUSSION TITLE
Pagination fragment (hash) support

### DISCUSSION THREAD

**Discussion by marekmez** [livewire/livewire#2874] [1. Help]
I have been trying to use Laravel Pagination fragment method to generate hash for each link in paginatior, but it seems not working with Livewire. Hash is just skiped (not attached).

I am using boostrap template.
`use WithPagination;
 protected $paginationTheme = 'bootstrap';`

Laravel Docs:
https://laravel.com/docs/8.x/pagination#appending-hash-fragments

Please do not hesitate to suggest any workaround or maybe I missed something in docs.

**Comment by Luigidefra** [comment]
Same problem, any suggest?
